client/coral-ui/components/Checkbox.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import styles from './Checkbox.css';
 
-export default ({name, cStyle = 'base', onChange, label, className, info, checked = 'false'}) => (
+export default ({name, cStyle = 'base', onChange, label, className, info, ...props}) => (
```

Comment:
Why pass in all ...props here? We have seen this pattern cause unexpected problems when things are changed upstream.